src/main/java/bio/terra/cli/app/CommandRunner.java

```diff
@@ -139,16 +140,24 @@ private Map<String, String> buildMapOfTerraReferences() {
                           // This should NEVER happen but check here to prevent endless resolve.
                           r -> Resource.Type.DATA_COLLECTION != r.getResourceType())
                       .forEach(
-                          r ->
-                              terraReferences.put(
-                                  "TERRA_" + resource.getName() + "_" + r.getName(), r.resolve()));
+                          r -> {
+                            String envVariable =
+                                convertToEnvironmentVariable(
+                                    "TERRA_" + resource.getName() + "_" + r.getName());
+                            terraReferences.put(envVariable, r.resolve());
+                          });
                 }
               }
             });
 
     return terraReferences;
   }
 
+  private static final String convertToEnvironmentVariable(String string) {
+    // Env variables can't have "-"
+    return string.replace("-", "_");
```

Comment:
oh sorry i forgot CLI didn't allow dashes because of environment variable.

but i don't think this would guarantee uniqueness of the env names. if there is a bq-1 and a bq_1, both of them will be TERRA_bq_1 envVariable.

Reply:
Replaced with double underscore instead of single underscore. Technically there could still be a conflict but chances are extremely low.

I'm concerned that double underscore is unintuitive -- people will try single underscore and wonder why it doesn't work. We'll see what happens.